pkg/virt-controller/watch/migration_test.go

```diff
@@ -149,8 +149,9 @@ var _ = Describe("Migration watcher", func() {
 			Expect(ok).To(BeTrue())
 			Expect(patch.GetPatchType()).To(Equal(types.StrategicMergePatchType))
 
-			expectedPatch := fmt.Sprintf(`{"spec":{"minAvailable": 2},"metadata":{"labels":{"%s": "%s"}}}`, virtv1.MigrationNameLabel, vmim.Name)
-			Expect(string(patch.GetPatch())).To(Equal(expectedPatch))
+			expectedPatch := fmt.Sprintf(`{"spec":{"minAvailable": 2},"metadata":{"labels":{"%s": "%s"`, virtv1.MigrationNameLabel, vmim.Name)
```

Comment:
It looks like using the prefix approach makes the test more relaxed comparing to what it was before when it expected explicit labels.